src/js/css-calc.ts

```diff
@@ -823,10 +892,13 @@ export const cssCalc = (value, opt = {}) => {
     throw new TypeError(`${value} is not a string`);
   }
   let cacheKey;
-  if (typeof dimension?.callback !== 'function') {
+  const _dimension: {
```

Comment:
If we are not going to use `?.` then change it to:
```javascript
if (!dimension || typeof dimension.callback !== 'function') {
}
```

Reply:
We are using `?.`